Components/Button/button.html

```diff
@@ -0,0 +1,173 @@
+                <li><a class="sidebar-item" href="">Colours</a></li>
+                <li>
+                    <h3>Components</h3>
+                </li>
+                <hr>
+                <li><a class="sidebar-item" href="/Components/Avatar/avatar.html">Avatar</a></li>
+                <li><a class="sidebar-item " href="/Components/Alert/alert.html">Alert</a></li>
+                <li><a class="sidebar-item " href="/Components/Badge/badge.html">Badge</a></li>
+                <li><a class="sidebar-item link-active" href="#">Button</a></li>
+                <li><a class="sidebar-item" href="/Components/Card/card.html">Card</a></li>
+                <li><a class="sidebar-item" href="/Components/Image/image.html">Image</a></li>
+                <li"><a class="sidebar-item" href="/Components/Input/input.html">Input</a></li>
+            </ul>
+        </div>
+        <section class="button-section">
+            <div class="section-title">
+                Button
+            </div>
+            <div class="section-subtitle">
+                Buttons are the soul of an interactive Web Application. Buttons allow users to take actions, and make
+                choices, with a single tap.
+            </div>
+            <hr>
+            <article>
+                <div class="article-title">
+                    Contained Button
+                </div>
+                <div class="article-subtitle">
+                    Contained buttons are high-emphasis, distinguished by their use of elevation and fill. They contain
+                    actions that are primary to your app
+                </div>
+                <div class="example-div">
+                    <button class="btn">DEFAULT</button>
+
+                    <button class="btn btn-primary">PRIMARY</button>
+
+                    <button class="btn btn-secondary">SECONDARY</button>
+
+                    <button class="btn btn-error">ERROR</button>
+
+                    <button class="btn btn-success">SUCCESS</button>
+                </div>
+                <div class="code-frame">
+                    <iframe
+                        src="https://carbon.now.sh/embed?bg=rgba%28184%2C176%2C141%2C0%29&t=3024-night&wt=none&l=auto&width=680&ds=true&dsyoff=0px&dsblur=68px&wc=true&wa=true&pv=56px&ph=56px&ln=false&fl=1&fm=Hack&fs=14px&lh=133%25&si=false&es=4x&wm=false&code=%253Cbutton%2520class%253D%2522btn%2522%253EDEFAULT%253C%252Fbutton%253E%250A%250A%253Cbutton%2520class%253D%2522btn%2520btn-primary%2522%253EPRIMARY%253C%252Fbutton%253E%250A%250A%253Cbutton%2520class%253D%2522btn%2520btn-secondary%2522%253ESECONDARY%253C%252Fbutton%253E%250A%250A%253Cbutton%2520class%253D%2522btn%2520btn-error%2522%253EERROR%253C%252Fbutton%253E%250A%250A%253Cbutton%2520class%253D%2522btn%2520btn-success%2522%253ESUCCESS%253C%252Fbutton%253E"
+                        style="width: 589px; height: 353px; border:0; transform: scale(1); overflow:hidden;"
```

Comment:
you should avoid using inline styles

Reply:
This is the default style from the iframe itself, should I have to change that ?